frontend/src/components/HomeSpeedDial.tsx

```diff
@@ -34,7 +34,7 @@ const HomeSpeedDial: React.FC<Props> = ({ onDownloadOpen, onEditorOpen }) => {
     >
       <SpeedDialAction
         icon={listView ? <ViewAgendaIcon /> : <FormatListBulleted />}
-        tooltipTitle={listView ? 'Card view' : 'Table view'}
+        tooltipTitle={listView ? 'Card view' : i18n.t('tableView')}
```

Comment:
IMHO, if we translate the second string, maybe we should also translate the first.